<launch>
    <node pkg="tf2_ros" type="static_transform_publisher" name="word_slam" args="0 0 0  0.5, -0.5, 0.5, -0.5  world slam" />
    <node pkg="tf2_ros" type="static_transform_publisher" name="zed_robot" args="0 0 0  0.5, -0.5, 0.5, 0.5  zed robot" />

    <arg name="prefix" default="" />
    <arg name="orb-dir" default="$(env HOME)/ORB_SLAM3" />

    <node pkg="disinfslam" type="ros_offline_orb3" name="ros_offline_orb3" output="screen" launch-prefix="$(arg prefix)"    >
        <param name="model_path" value="/home/yu/tools/ht_lt.pt" />
        <param name="calib_path " value="$(find disinfslam)/configs/zed_28498913_orb3.yaml" />
        <param name="orb_vocab_path" value="$(arg orb-dir)/Vocabulary/ORBvoc.txt" />
        <param name="renderer" type="bool" value="false" />
        <param name="use_mask" type="bool" value="false" />
        <param name="bbox_xy" type="double" value="8.0" />
        <param name="sensor" type="int" value="4" />
        <remap from="/stereoLeft" to="/zed2/zed_node/left_raw/image_raw_color" />
        <remap from="/stereoRight" to="/zed2/zed_node/right_raw/image_raw_color" />
        <remap from="/depth" to="/camera/aligned_depth_to_color/image_raw" />
        <remap from="/rgbImg" to="/camera/color/image_raw" />

    </node>
</launch>

<!-- cuda-memcheck -\-log-file /home/motion/disinfect/cuda.log -->
<!-- cuda-gdb -->
<!-- gdb -ex run -\-args -->